service/service.go

```diff
@@ -43,10 +43,6 @@ type service struct {
 }
 
 func newService(set *svcSettings) (*service, error) {
-	if err := set.Config.Validate(); err != nil {
-		return nil, fmt.Errorf("invalid configuration: %w", err)
-	}
-
```

Comment:
Is this a duplicate? Can you please point where it's being called?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/blob/main/service/config_watcher.go#L49

Reply:
Thanks